### PR TITLE
Feature/dev 102

### PR DESCRIPTION
**Pull request by Himansh1706**

### CODE REVIEW

File: Backend/userservice/.mvn/wrapper/maven-wrapper.properties
Comment (iamdexter12) [inline Backend/userservice/.mvn/wrapper/maven-wrapper.properties:1]: remove all this unneccessary files

File: Backend/userservice/pom.xml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+		</dependency>
+		<dependency>
+			<groupId>io.springfox</groupId>
+			<artifactId>springfox-swagger2</artifactId>
+			<version>3.0.0</version>
+		</dependency>
+		<dependency>
+			<groupId>io.springfox</groupId>
+			<artifactId>springfox-swagger-ui</artifactId>
+			<version>3.0.0</version>
```
Comment (iamdexter12) [inline Backend/userservice/pom.xml]: Remove unneccssary dependency

File: Backend/userservice/src/main/java/com/cbh/userservice/requestdto/AddUser.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import lombok.NoArgsConstructor;
+
+/**
+ * Data class representing the information needed to add a new user.
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class AddUser {
+	private String name;
```
Comment (iamdexter12) [inline Backend/userservice/src/main/java/com/cbh/userservice/requestdto/AddUser.java]: Please correct formatting

File: Backend/userservice/src/main/java/com/cbh/userservice/requestdto/UpdateUser.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class UpdateUser {
+	
+	private String name;
```
Comment (iamdexter12) [inline Backend/userservice/src/main/java/com/cbh/userservice/requestdto/UpdateUser.java]: Same

**Comment by iamdexter12** [comment]
Nice work, Have few comments do check

File: Backend/userservice/pom.xml (hunk 1 above)
Comment (Himansh1706) [reply Backend/userservice/pom.xml]: done

File: Backend/userservice/src/main/java/com/cbh/userservice/requestdto/AddUser.java (hunk 2 above)
Comment (Himansh1706) [reply Backend/userservice/src/main/java/com/cbh/userservice/requestdto/AddUser.java]: done

File: Backend/userservice/src/main/java/com/cbh/userservice/requestdto/UpdateUser.java (hunk 3 above)
Comment (Himansh1706) [reply Backend/userservice/src/main/java/com/cbh/userservice/requestdto/UpdateUser.java]: done

**Pull request merged**